substrate/frame/recovery/src/benchmarking.rs

```diff
@@ -353,17 +329,14 @@ benchmarks! {
 
 		// Reserve deposit for recovery
 		T::Currency::reserve(&caller, total_deposit).unwrap();
-	}: _(
-		RawOrigin::Signed(caller.clone())
-	) verify {
-		assert_last_event::<T>(
-			Event::RecoveryRemoved {
-				lost_account: caller
-			}.into()
-		);
+
+		#[extrinsic_call]
+		_(RawOrigin::Signed(caller.clone()));
+		assert_last_event::<T>(Event::RecoveryRemoved { lost_account: caller }.into());
 	}
 
-	cancel_recovered {
+	#[benchmark]
+	fn cancel_recovered() -> Result<(), BenchmarkError> {
```

Comment:
I think if you dont use a `?` then you can also remove the return type and just return `()`.

Reply:
I just preserved the previous behavior, but happy to change it. I was wondering if there is any good use for `BenchmarkError` at all.

Reply:
Yea its fine either way. Just to know for the next time 😉